s3-fallback-store/src/main/resources/reference.conf

```diff
@@ -0,0 +1,122 @@
+// #fallback-settings
+akka.persistence.s3-fallback-store {
+  class = "akka.persistence.s3fallback.S3Fallback"
+
+  # Absolute config path for settings for the Netty NIO http client.
+  #
+  # See 'http-defaults' for the structure.
+  #
+  # If using the DynamoDB plugin, you may point it to the same settings
+  http-client-config = "akka.persistence.s3-fallback-store.http-defaults"
+
+  # The S3 bucket in which to store large events: must be overridden
+  # to enable storing events
+  events-bucket = ""
+
+  # The S3 bucket in which to store large snapshots: must be overridden
+  # to enable storing snapshots
+  snapshots-bucket = ""
+
+  # Can override the region if desired
+  region = ""
+
+  # Multipart settings
+  #
+  # Multipart is enabled only if set to an object of the form
+  #
+  # multipart {
+  #   # Size threshold to use multipart uploads/downloads
+  #   # Minimum allowed by S3 is 5 MiB
+  #   # Set to Long.MAX_VALUE/Long.MaxValue to disable
+  #   threshold = 5 MiB
```

Comment:
Maybe support setting to `off` or `none` strings to disable in config. Internally can use `Long.MaxValue`.